Lib/collections/__init__.py

```diff
@@ -38,6 +38,8 @@
 except ImportError:
     pass
 
+_GenericAlias = type(list[int])
```

Comment:
Is this supposed to be here?  AFAICT it isn't referenced anywhere and the tests pass without it.